circle.yml

```diff
@@ -23,7 +23,7 @@ machine:
     - redis
 checkout:
   post:
-    - case $CIRCLE_NODE_INDEX in 1) ./bin/circle/merge-eyes-baselines ;; *) echo "Skipping step." ;; esac:
+    - ./bin/circle/merge-eyes-baselines:
```

Comment:
Leaving this here to keep the dream alive.  The baseline merge is actually disabled [within the `merge-eyes-baselies` script](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/circle/merge-eyes-baselines#L6-L7).